src/includes/index.ts

```diff
@@ -0,0 +1,26 @@
+import { combine, Store } from 'effector';
+
+export function includes <T extends string>(
+  a: Store<T>,
+  b: Store<T> | T,
+): Store<boolean>
+export function includes <T extends string | number>(
+  a: Store<Array<T>>,
+  b: Store<T> | T,
+): Store<boolean>
```

Comment:
What if:

```suggestion
export function includes <T extends string>(
  a: Store<T>,
  b: Store<T> | T,
): Store<boolean>
export function includes <T>(
  a: Store<Array<T>>,
  b: Store<T> | T,
): Store<boolean>
export function includes <T>(
  a: Store<{ ref: Set<T> }>,
  b: Store<T> | T,
): Store<boolean>
```

allow to compare with any list of items?